changelog.md

```diff
@@ -8,10 +8,12 @@
 * You can now add two `KymoTrackGroups` tracked on the same kymo together with the `+` operator.
 * TIFFs exported from `Scan` and `Kymo` now contain metadata. The `DateTime` tag indicated the start/stop timestamp of each frame. The `ImageDescription` tag contains additional information about the confocal acquisition parameters.
 * Added covariance-based estimator (cve) option to `KymoTrack.estimate_diffusion`. See [kymotracker documentation](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/kymotracking.html#studying-diffusion-processes) for more details.
+* Added the optional `min_length` argument to `KymoTrackGroup.estimate_diffusion` to discard tracks shorter than a specified length from the analysis.
```

Comment:
What you added was the optional parameter to the function _signature_. The argument is how the function is called.

Reply:
How would you word this then? I know it's more technically correct, so far this is the way we've been describing these kinds of changes in the changelog but I'd be ok changing if it sounds ok. Maybe
```suggestion
* Added the optional `min_length` parameter to the `KymoTrackGroup.estimate_diffusion()` signature. Use this parameter to discard tracks shorter than a specified length from the analysis.
```